packages/router-devtools/src/index.tsx

```diff
@@ -1,2 +1,6 @@
+console.warn(
+  '[@tanstack/router-devtools] This package has moved to @tanstack/react-router-devtools. Please switch to the new package at your earliest convenience, as this package will be dropped in the next major version release.',
+)
+
 export { TanStackRouterDevtools } from './TanStackRouterDevtools'
```

Comment:
Has the implementation details (components, utils, etc.) for `@tanstack/router-devtools` not been nuked yet?

---

To retain the same "available in production" behaviour, maybe just export a legacy option from `@tanstack/react-router-devtools` and have that be the primary export for `@tanstack/router-devtools`.

So, that'd be.

1. Setup an export as `@tanstack/react-router-devtools/legacy`.
1.1 Legacy export would retain the same behaviour in deployment as we have now.
1.2 `packages/react-router-devtools/vite.config.ts` to have a separate entry for the legacy entry.
1.3 `packages/react-router-devtools/package.json` to have a separate entry for the legacy entry.
2. `@tanstack/router-devtools` exports the components from `@tanstack/react-router-devtools/legacy`.

Reply:
We just need to maintain the package export for `@tanstack/router-devtools`. The package itself would basically just be proxying out `@tanstack/react-router-devtool/legacy`.

Essentially, the same as what we are doing for `@tanstack/router-vite-plugin`.

Reply:
Having the `@tanstack/router-devtools` 

`export { TanStackRouterDevtoolsInProd as TanStackRouterDevtools } from '@tanstack/react-router-devtools'` 

Will retain the existing  beahvior

Reply:
Awesome!

I just don't want to maintain any implementation in `@tanstack/router-devtools`. It should all just live in `@tanstack/react-router-devtools`.

The `@tanstack/router-devtools` would just be a proxy. No implementation.

Reply:
I've removed the eimplementation, so it's just a proxy now, as demonstrated in `/e2e/react-router/basic-react-query`